embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/SessionHandler.kt

```diff
@@ -48,6 +48,32 @@ internal class SessionHandler(
     private val sessionPeriodicCacheExecutorService: ScheduledExecutorService
 ) : Closeable {
 
+    /**
+     * Defines the states in which a session can end.
+     */
+    private enum class SessionEndType(
+        val endedCleanly: Boolean,
```

Comment:
Do we need to add a comment about what each of these properties mean or will it be obvious at their usage points?

Reply:
Added comments